test/runtests.jl

```diff
@@ -5,3 +5,10 @@ for t in tests
     println("* running $fp")
     include(fp)
 end
+
+if VERSION >= v"1.7"
+    using JET
+    @testset "Static analysis" begin
+        @test isempty(JET.get_reports(report_package("StatsFuns", target_modules=(StatsFuns,))))
```

Comment:
One could also use `JET.test_package`, e.g.,
```suggestion
        JET.test_package("StatsFuns", target_defined_modules=true)
```

Reply:
Or
```suggestion
        JET.test_package("StatsFuns", target_defined_modules=true)
```